app/controllers/merchants/base_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Merchants::BaseController < ApplicationController
+  before_action :require_merchant
+
+  def require_merchant
+    if !current_merchant?  
+      render file: "/public/404"
```

Comment:
Love that this worked.

Reply:
💯 me too! In future stories we'll figure out how get these tests written properly.